### PR TITLE
Update PyPI version number

### PR DESCRIPTION
**Pull request by apeforest**
Update PyPI version number to refresh the PyPI package.

### CODE REVIEW

**Comment by apeforest** [comment]
@pracheer Please review this as we discussed. Thanks!

File: tools/coreml/pip_package/setup.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def readme():
         return f.read()
 
 setup(name='mxnet-to-coreml',
-      version='0.1.0',
+      version='0.1.1',
```
Comment (pracheer) [inline tools/coreml/pip_package/setup.py]: https://pypi.org/project/mxnet-to-coreml/ seems to indicate we are already at 0.1.2.
Update this to 0.1.3 instead?

Comment (apeforest) [reply tools/coreml/pip_package/setup.py]: Updated

**Comment by pracheer** [comment]
Looks good to me! Thanks @apeforest

**Comment by sandeep-krishnamurthy** [comment]
@apeforest - Thank you for your contributions. Can you re-trigger the build?

**Comment by sandeep-krishnamurthy** [comment]
@apeforest - ping.

**Comment by apeforest** [comment]
@sandeep-krishnamurthy re-build. Please approve. Thanks!

**Comment by lupesko** [comment]
@sandeep-krishnamurthy @nswamy @anirudh2290 - kindly review and merge if looks OK. Thanks!

**Pull request merged**